tests/conformance/conformance_test.go

```diff
@@ -952,9 +957,18 @@ func summarizeLayer(t *testing.T, imageName string, blobInfo types.BlobInfo, rea
 		layer.Headers = append(layer.Headers, header)
 		hdr, err = tr.Next()
 	}
+	io.Copy(io.Discard, tarToRead)
```

Comment:
Consider:
- Moving this down one line, to keep the code setting `err` and checking it together
- Checking for a failure of this `io.Copy` as well.

Reply:
… a linter is complaining about the latter part, failing the build.

Reply:
I had wondered if a linter would complain about the second one.  Changed.